x-pack/spec/modules/arcsight/yaml/event_broker_with_security.yml

```diff
@@ -0,0 +1,17 @@
+modules:
+  - name: arcsight
+    var.inputs: "kafka"
```

Comment:
Would it make sense to use `eventbroker` rather than `kafka` to be consistent with filename and documentation?

Reply:
The conversion from `eventbroker` to `kafka` is done by the ERB file before the module methods are called. The module method is expecting `kafka`. I suppose the module method could test and switch upon either one. Then the file and contents could be changed.
Alternatively, would a rename of the file satisfy?

Reply:
I see. That makes sense. 

Maybe a comment would suffice

Reply:
Added comments to yaml